packages/avatars/src/styled/StyledStandaloneStatus.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'avatars.status-indicator';
+
+export const StyledStandaloneStatus = styled.figure.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<ThemeProps<DefaultTheme>>`
+  box-sizing: content-box;
+  display: inline-flex;
+  flex-flow: row wrap;
```

Comment:
The flex layout here is not to design spec. The indicator should remain in its own column when the text has to wrap.

<img width="288" alt="Screen Shot 2022-09-15 at 8 55 10 AM" src="https://user-images.githubusercontent.com/143773/190413941-6e597613-6a26-45db-9d45-93deba7c95f4.png">

Reply:
Updated this in [ad377d7](https://github.com/zendeskgarden/react-components/pull/1410/commits/ad377d717954dc8b815886e55db728987238edad).